src/devTools.js

```diff
@@ -257,8 +257,29 @@ function handleChange(state, liftedState, maxAge) {
   }
 }
 
+/*
+ * Get hostname for React Native if hostname is `localhost`
+ * For Android Emulator, the real localhost is `10.0.2.2` (Genymotion: 10.0.3.2)
+ */
+function getHostForRN(hostname) {
+  if (hostname !== 'localhost' || hostname !== '127.0.0.1') {
+    return hostname;
+  }
+
+  let host = hostname;
+  try {
```

Comment:
Done.

Although it just a little affect for this case, but I think it's a bad habits. :D